dist/scss/wds-components/_dropdowns.scss

```diff
@@ -1,5 +1,9 @@
+$border-radius: 4px;
+$dropdown-border-width: 1px;
```

Comment:
I guess these variables should be internal. Or if not - they should have 'wds-' prefix.